MSB6004: The specified task executable location is invalid.

Build FAILED.

/home/gitpod/dotnet/sdk/5.0.205/Roslyn/Microsoft.CSharp.Core.targets(71,5): error MSB6004: The specified task executable location "/run/containerd/io.containerd.runtime.v2.task/k8s.io/252e8970cc03942c895c7462c53a2aecd606d5f8c9d73ee3e4a93a765654ff03/rootfs/home/gitpod/dotnet/dotnet" is invalid. [/workspace/template-dotnet-core-cli-csharp/dotnetcore.csproj]
    0 Warning(s)
    1 Error(s)

--- a/.gitpod.Dockerfile
+++ b/.gitpod.Dockerfile
@@ -2,5 +2,6 @@ FROM gitpod/workspace-full:latest
 
 USER gitpod
 #.NET installed via .gitpod.yml task until the following issue is fixed: https://github.com/gitpod-io/gitpod/issues/5090
-ENV DOTNET_ROOT=/tmp/dotnet
-ENV PATH=$PATH:/tmp/dotnet
+ENV DOTNET_VERSION=6.0
+ENV DOTNET_ROOT=/workspace/.dotnet
+ENV PATH=$PATH:$DOTNET_ROOT

--- a/.gitpod.yml
+++ b/.gitpod.yml
@@ -5,7 +5,7 @@ tasks:
   # Mitigation for https://github.com/gitpod-io/gitpod/issues/6460 
   - name: Postinstall .NET 6.0 and dev certificates
     init: |
-      mkdir -p /tmp/dotnet && curl -fsSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --channel 6.0 --install-dir /tmp/dotnet 
+      mkdir -p $DOTNET_ROOT && curl -fsSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --channel $DOTNET_VERSION --install-dir $DOTNET_ROOT 
       dotnet dev-certs https 
       dotnet restore
 

